pkg/util/ciutil/systems/az_pipelines.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016-2019, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package systems
+
+import (
+	"fmt"
+	"os"
+)
+
+// AzurePipelinesCISystem represents the Azure Pipelines CI/CD system
+// that belongs to the Azure DevOps product suite.
+type AzurePipelinesCISystem struct {
```

Comment:
Please remove `System` from every exported type from the `systems` package, as it "stutters". (Though in this case, via a suffix rather than prefix, which is probably why it wasn't caught by a linter.)

https://blog.golang.org/package-names
> Avoid stutter. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the http package is called Server, not HTTPServer. Client code refers to this type as http.Server, so there is no ambiguity.